src/citrine/informatics/workflows/design_workflow.py

```diff
@@ -33,27 +34,16 @@ class DesignWorkflow(Resource['DesignWorkflow'], Workflow):
 
     uid = properties.Optional(properties.UUID, 'id', serializable=False)
     name = properties.String('display_name')
```

Comment:
`name` is implemented twice

Reply:
the two different `name`s are for the old style and new style workflows. `display_name` was used by the module (old) and `name` is used by the DW (new). The overwriting is weird, but not a change introduced in this pr